### PR TITLE
Basic Testing

### PR DESCRIPTION
**Pull request by sungho-cho**
- Reorganize the project
- Write tests for Poll model

### CODE REVIEW

File: CMUPollTests/PollTests.swift [hunk 1]
```diff
@@ -170,4 +170,77 @@ class PollTests: XCTestCase {
...
+  func testWithTag1() {
+    let expectation = self.expectation(description: "Test withTag1")
+    Poll.withTag(name: "tag that does not exist") { polls in
+      XCTAssertEqual(0, polls.count)
+      expectation.fulfill()
+    }
+    self.waitForExpectations(timeout: 5.0, handler: nil)
+  }
+  
+  func testCreateUpdateDelete() {
```
Comment (leesihoney) [inline CMUPollTests/PollTests.swift:223]: 여기 잘돼?

**Review by leesihoney — COMMENTED** [review]
테스팅 잘됨?

**Comment by AidenLee94** [comment]
GOOOD!

**Comment by leesihoney** [comment]
GOOOD

**Pull request merged**